feat(IAST): Added the entire section (#14569)
newrelic/docs-website: src/content/docs/iast/summary.mdx

---
title: New Relic IAST summary page
tags:
    - New Relic IAST
    - IAST
metaDescription: Find out the data you can find on the New Relic IAST summary page.
---

The New Relic IAST summary page provides a centralized view of your tested applications and your vulnerabilities at a glance. View vulnerabilities and their risk scores helps you to prioritize remediation decisions.

Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > IAST** to open the New Relic IAST summary page.

> <img

Reviewer: I worry that it's too difficult to parse this image. Maybe something like the labeled image here? https://docs.newrelic.com/docs/infrastructure/infrastructure-ui-pages/infra-ui-overview/

    title="New Relic IAST Summary page"
    alt="New Relic IAST Summary page"
    src={iastSummary}
/>

> <figcaption>

Reviewer: There's a discrepancy between what's in the caption and what's in the path up above.

   Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**.
</figcaption> 

These are the different parts of the summary page:

- **Exploitable vulnerabilities by severity**. Check out a summary of the number of vulnerabilities you have by the [CVSS severity score](/docs/iast/overview/#vuln-severity): critical, high, and the total number of vulnerabilities. 

- **Exploitable categories detected**. This is an ordered list of the most exploitable categories detected.

- **Applications detected during testing**. Take a glance at the number of vulnerable applications you have.

> - **IAST coverage**. Pie chart for having a view of the total number of applications and the number of tested and untested applications.

Reviewer: ```suggestion
- **IAST coverage**. Pie chart with a view of the total number of applications and the number of tested and untested applications.
```

> - **Take these actions**. This table shows you a list of the most exploitable applications. Click on an application to open its specific vulnerability page. There is also a list of the untested applications. Click on an untested application if you want to [enable IAST](/docs/iast/applications/#enable-iast).

Reviewer: "There is also a list of untested applications."

Maybe make these two sentences their own bullet? Feels like a separate thing.
